frontend/ui/uimanager.lua

```diff
@@ -116,8 +116,7 @@ function UIManager:init()
         self.event_handlers["Resume"] = function()
             self:_afterResume()
         end
-    end
-    if Device:isKobo() then
+    elseif Device:isKobo() then
```

Comment:
I'd prefer to resolve the nit alphabetically, if we're going to be touching some lines for nit reasons anyway.

Reply:
Alphabetically seem fair and even more readable than the current code. But the diff will blow up.

Reply:
Whoa, I only meant the PB one, not as in diff everything. :o